x/pool/types/keys.go

```diff
@@ -0,0 +1,15 @@
+package types
+
+const (
+	// ModuleName is the module name constant used in many places
+	ModuleName = "pool"
```

Comment:
should we use something like comsos-sdk-pool. I think others have modules that are called pool, this would conflict with those